plat/nvidia/tegra/include/tegra_private.h

```diff
@@ -53,6 +53,11 @@ typedef struct plat_params_from_bl2 {
  ******************************************************************************/
 DEFINE_RENAME_SYSREG_RW_FUNCS(l2ctlr_el1, CORTEX_A57_L2CTLR_EL1)
 
+/*******************************************************************************
+ * Helper function to access id_afr0_el1 (Auxiliary Feature Register 0) register
+ ******************************************************************************/
+DEFINE_SYSREG_READ_FUNC(id_afr0_el1)
```

Comment:
I think this should be in ``include/arch/aarch64/arch_helpers.h`` so that any platform can use it.